test/unittests/libweb3jsonrpc/jsonrpc.cpp

```diff
@@ -377,8 +437,9 @@ BOOST_AUTO_TEST_CASE(eth_sendRawTransaction_errorInvalidNonce)
     auto txHash = rpcClient->eth_sendRawTransaction(signedTx["raw"].asString());
     BOOST_REQUIRE(!txHash.empty());
 
-    auto invalidNonce =
-        jsToU256(rpcClient->eth_getTransactionCount(toJS(senderAddress), "latest")) - 1;
```

Comment:
without mining previously sent transaction, `getTransactionCount` here returned 0, then u256 underflowed, and `invalidNonce` was max<u256>

I replaced it with just using 0 as invalid nonce

Reply:
Ah, nice catch!